sourcecode/hub/resources/css/theme.scss

```diff
@@ -13,4 +13,35 @@
     .navbar-bottom {
         background-color: $green-200;
     }
+
+    p, h1 {
```

Comment:
These elements should not specifically have this font. The prototype uses Helvetica everywhere, so the default font should be changed via the appropriate variable in Bootstrap.

Additionally, Helvetica Neue is only available on Apple devices, so this needs a fallback.